scripts/create_test_subset.py

```diff
@@ -137,7 +172,19 @@ def main(
 
     else:
         assert samples_n
```

Comment:
Not related to this change, but better to check and raise with a specific message. Also `asserts` can be skipped when running with Python in some modes (w/ `-O`).